### ISSUE TITLE
Provided schema doesn't exist for path with space in name

### ISSUE THREAD

**Issue by DevanB** [redwoodjs/redwood#5450]
Hi there! Sometime recently I upgraded from 1.1 to 1.3. After doing this (painlessly), I noticed an error when I would run `yarn rw prisma migrate dev` in the form of:

```bash
Error: Provided --schema at /Users/devanb/My doesn't exist.
```

<details>
 <summary>The full error can be seen here</summary>

```bash
21:24 $ yarn rw prisma migrate dev
(node:48782) [FST_MODULE_DEP_FASTIFY-REPLY-FROM] FastifyWarning.fastify-reply-from: fastify-reply-from has been deprecated. Use @fastify/reply-from@7.0.0 instead.
(Use `node --trace-warnings ...` to show where the warning was created)

Running Prisma CLI...
$ yarn prisma migrate dev --schema ""/Users/dbeitel/My Apps/my-rw-app/api/db/schema.prisma""

Environment variables loaded from .env
Error: Provided --schema at /Users/dbeitel/My doesn't exist.
```
</details>

Since the initial time I got the error, I had changed my parent folder to not have a space within it and went on my way. However, over the last few days, I've seen others recording this issue on Discord as well. So I decided to create this quick walk-through to create a minimal reproduction:

1. In the terminal run `mkdir -p My\ Apps && cd My\ Apps`
2. After creating and changing into the directory, execute `yarn create redwood-app my-rw-app --typescript`
3. Once the project setup is complete, change into the new project with `cd my-rw-app`
4. Finally, execute the Prisma migrate command with `yarn rw prisma migrate dev`

Assuming all goes as (unfortunately) planned, you should be presented with an error very similar to the one at the beginning of this post (within the details accordion). 

I'll be glad to assist as much as possible, but wouldn't know where to begin to debug this issue.

**Comment by jtoar** [comment]
@DevanB Thanks for the write up! I can reproduce this. And I think I've narrowed it down to the change I made here: https://github.com/redwoodjs/redwood/pull/5147. The debugger shows the problem pretty clearly:

<img width="1522" alt="image" src="https://user-images.githubusercontent.com/32992335/167081094-14c41ceb-b9c6-43b7-ab21-01da22d18a85.png">

I made the change to fix this issue: https://github.com/redwoodjs/redwood/issues/5135.

Would you be up for trying to fix this one now that we think we know where things are going wrong?

**Comment by dac09** [comment]
Could we make sure there’s a unit test for this when the fix is in place please?

**Comment by Isaac-Tait** [comment]
For posterity sake I am sharing the forum post I created when I ran into this same issue [here](https://community.redwoodjs.com/t/prisma-migrate-error/3196) Replacing the space in the folder name with an underscore `_` alleviated the issue.

**Comment by jtoar** [comment]
@DevanB @Isaac-Tait this is now patched in v1.3.3: https://github.com/redwoodjs/redwood/releases/tag/v1.3.3.
